roles/lnls-ans-role-sirius-hla/tasks/main.yml

```diff
@@ -0,0 +1,58 @@
+---
+- name: Resolve platform specific vars
+  include_vars: '{{ item }}'
+  with_first_found:
+    - files:
+        - '{{ ansible_distribution }}-{{ ansible_distribution_release }}.yml'
+        - '{{ ansible_distribution }}.yml'
+        - '{{ ansible_os_family }}.yml'
+      ignore_errors: true
+      paths:
+        - '{{ role_path }}/vars'
+
+- name: Define Python executable
+  set_fact:
+    python_executable: "{{ __python_executable | default('python') }}"
+  when: python_executable is not defined
+
+- name: Define PIP executable
+  set_fact:
+    pip_executable: "{{ __pip_executable | default(omit) }}"
+  when: pip_executable is not defined
+
+- name: Define sirius packages
+  set_fact:
+    sirius_apps_packages: "{{ __sirius_apps_packages | list }}"
+  when: sirius_apps_packages is not defined
+
+- name: Define sirius pip packages
+  set_fact:
+    sirius_apps_pip_packages: "{{ __sirius_apps_pip_packages | list }}"
+  when: sirius_apps_pip_packages is not defined
+
+# Combine apps to install
+- name: Create list of dict apps to install
+  set_fact:
+    sirius_apps: "{{ sirius_apps | default([]) |
```

Comment:
Wrong variable name, should be `hla_apps`

Reply:
done